portals/admin/source/src/app/components/APICategories/AddEditAPICategory.jsx

```diff
@@ -113,6 +119,12 @@ function AddEdit(props) {
         if (NameErrors) {
             errorText += NameErrors + '\n';
         }
+        if (description != undefined) {
```

Comment:
@Sachini-Dissanayaka seems there are some build issue here. check the build jobs logs
<img width="863" alt="Screenshot 2022-08-15 at 12 08 28 PM" src="https://user-images.githubusercontent.com/4861150/184588243-2f0316e0-2354-403e-8d48-7d8edbb1a1e7.png">